gulp/tasks/scripts.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+var gulp = require('gulp');
+var runSequence = require('run-sequence');
+var filenames = require('gulp-filenames');
+var concat = require('gulp-concat');
+var closureCompiler = require('gulp-closure-compiler');
+var uglify = require('gulp-uglify');
+var config = require('../config');
+
+gulp.task('compile_js', function() {
+  var closureOpts = {
+    compilerPath: './bower_components/closure-compiler/compiler.jar',
+    compilerFlags: {
+        angular_pass: true,
+        closure_entry_point: 'scaffold',
+        compilation_level: 'SIMPLE_OPTIMIZATIONS',
+        generate_exports: true,
+        manage_closure_dependencies: true,
+        only_closure_dependencies: true,
+        output_wrapper: '(function(){%output%})();',
+        js: [
+            './bower_components/closure-library/closure/**.js',
+            './bower_components/closure-library/third_party/**.js',
+            '!**_test.js'
+        ]
+    },
+    maxBuffer: 800000, // Set maxBuffer to .8GB 
+    fileName: 'build.min.js'
+  };
+
+  var externs = [];
+  externs.concat(filenames.get('closure_externs'));
```

Comment:
I guess I just don't understand the magic behind `filenames('closure_externs')`. Or is this working because you have a task named `closure_externs` and also a variable named `closure_externs` (the argument to `filenames`)?

Is it possible to remove the closure_externs task entirely and just replace this line with something like `gulp.src(config.Path.CLOSURE_EXTERNS)`?